tags/guide/mixinthis.ytag

```diff
@@ -0,0 +1,5 @@
+type: text
+
+---
+
+`(({{0}}) (Object) this)`
```

Comment:
Could you add some description to say when this type of syntax would be used?

Reply:
Done!